src/themes/default/components/core/blocks/Auth/Login.vue

```diff
@@ -131,23 +144,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-  @import '../../../../css/text.scss';
-
-  input[type=password], input[type=email], input[type=text] {
-    @extend body;
+  input[type=password], input[type=email] {
     box-sizing: border-box;
-    font-size: 18px;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    border: none;
-    border-bottom: 1px solid #BDBDBD;
     width: 100%;
```

Comment:
We can make width/max-width 100% atomic also - they are used in many places

Reply:
Done